ml-agents/mlagents/trainers/ppo/policy.py

```diff
@@ -33,11 +37,28 @@ def __init__(self, seed, brain, trainer_params, is_training, load):
                 use_recurrent=trainer_params["use_recurrent"],
                 num_layers=int(trainer_params["num_layers"]),
                 m_size=self.m_size,
-                use_curiosity=bool(trainer_params["use_curiosity"]),
-                curiosity_strength=float(trainer_params["curiosity_strength"]),
-                curiosity_enc_size=float(trainer_params["curiosity_enc_size"]),
                 seed=seed,
+                stream_names=list(reward_signal_configs.keys()),
             )
+            self.model.create_ppo_optimizer()
+
+            # Create reward signals
+            for _rsignal in reward_signal_configs.keys():
```

Comment:
nit: `signal_name` instead of `_rsignal`?

Reply:
even better:
```
for signal_name, config in reward_signal_configs.items():
    self.reward_signals[signal_name] = create_reward_signal(self, signal_name, config)

Reply:
Done!